### DISCUSSION TITLE
Multiple movies on one disc

### DISCUSSION THREAD

**Discussion by thompcha** [automatic-ripping-machine/automatic-ripping-machine#630] [Q&A]
How does ARM handle multiple movies on a single disc?

**Comment by 1337-server** [comment]
I have no idea, it would depend on the disc. But the default is to assume it's a data disc and then use best guesses to find the correct type.

It also depends on the settings in the arm.yaml, with some settings it will only rip the largest movie file i can find, in others it will rip everything and put everything in the same folder.

I haven't run into this, so it's not something that has had any testing.
